securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -3,11 +3,13 @@
 
 {project_name} comes with a client-side JavaScript library that can be used to secure HTML5/JavaScript applications. The JavaScript adapter has built-in support for Cordova applications.
 
-The library can be retrieved directly from the {project_name} server at `{kc_js_path}/keycloak.js` and is also distributed as a ZIP archive.
+A good practice is to include the JavaScript adapter in your application using a javascript package manager like npm or yarn. The keycloak-js package is available on the following locations:
 
-A best practice is to load the JavaScript adapter directly from {project_name} Server as it will automatically be updated when you upgrade the server. If you copy the adapter to your web application instead, make sure you upgrade the adapter only after you have upgraded the server.
+* NPM: https://www.npmjs.com/package/keycloak-js
+* Yarn: https://yarnpkg.com/package/keycloak-js
+
+Alternatively, the library can be retrieved directly from the {project_name} server at `{kc_js_path}/keycloak.js` and is also distributed as a ZIP archive.
```

Comment:
It would be good to add a note that it is not recommended to load load the JavaScript from the server, as it might change at any point, possibly causing application code to misbehave.

Reply:
.. but feel free to suggest something, i'll add :)

Reply:
Yeah I'd say it's fine like this then. We do indeed tie to the Keycloak version with the NPM releases.